public/app/plugins/datasource/influxdb/influx_query_model.ts

```diff
@@ -20,6 +20,7 @@ export default class InfluxQueryModel {
     this.templateSrv = templateSrv;
     this.scopedVars = scopedVars;
 
+    target.policy = target.policy || 'default';
```

Comment:
can we clean up these string constants? Also we probably want null coalesce `??` instead of boolean OR here?

Reply:
looks like DEFAULT_POLICY is already available